### PR TITLE
docs: Added screenshots to the README

### LINKED ISSUE #11: Add screenshots to the README
PR Activity ref: #8 

Links to get started:
- https://www.educative.io/answers/adding-images-to-readmemd-in-github

### PR DESCRIPTION
**Pull request by yr1404**
Fixes #11

### CODE REVIEW

**Comment by yr1404** [comment]
@vijay-jaisankar Any changes required in this PR?

**Comment by vijay-jaisankar** [comment]
@yr1404 To keep up with accessibility standards, the docstrings and alt tags for images must be consistent - in this regard, there are a couple of (small) changes in this PR, please take a look at them in the ```Reviews``` section.

**Comment by yr1404** [comment]
> @yr1404 To keep up with accessibility standards, the docstrings and alt tags for images must be consistent - in this regard, there are a couple of (small) changes in this PR, please take a look at them in the `Reviews` section.

I am not able to find where are the changes listed. Can you help me with specifying those changes?

File: README.md [hunk 1]
```diff
@@ -28,3 +28,17 @@ pip install -r REQUIREMENTS.txt
 
 ## Running the application
 ```python app.py``` will run the application locally, it can be accessed by navigating to `http://127.0.0.1:5000`.
+
+---
+
+## Screenshots  
+
+![Screenshot1](https://github.com/yr1404/pandemonium/assets/106465753/70526657-b3d9-4a6a-a9e5-5f2aa8b68b65)
```
Comment (yr1404) [inline README.md:36]: What should the docstrings and alt tags be so that it follows the standards?

**Comment by yr1404** [comment]
@vijay-jaisankar I have made alt tags and docstrings consistent for the screenshots. Kindly review.

**Comment by vijay-jaisankar** [comment]
LGTM! Thanks.

**Pull request merged**